ui-admin/src/study/participants/merge/ParticipantMergeView.tsx

```diff
@@ -0,0 +1,232 @@
+            {renderMergeUserInfo(mergePlan, true)}
+            {mergeArrow}
+            {renderMergeUserInfo(mergePlan, false)}
+          </div>
+          <div>
+            { mergePlan.enrollees.map((merge, index) => <EnrolleeMergePlanView
+              enrolleeMerge={merge} key={index} studyEnvParams={paramsFromContext(studyEnvContext)}/>) }
+          </div>
+          <Button variant="primary" onClick={executeMerge} className="mt-3">Execute Merge</Button>
+        </div> }
+      </LoadingSpinner>
+    </div>
+  </div>
+}
+
+function EnrolleeMergePlanView({ enrolleeMerge,  studyEnvParams }:
+  {enrolleeMerge: MergeAction<Enrollee, EnrolleeMergePlan>, studyEnvParams: StudyEnvParams }) {
+  const sortedTasks = enrolleeMerge.mergePlan.tasks.sort((a, b) =>
+    taskComparator(a.pair.source ?? a.pair.target, b.pair.source ?? b.pair.target))
+  return <div className="mx-4">
+    { enrolleeMerge.pair.source &&
+      <Link to={studyEnvParticipantPath(studyEnvParams, enrolleeMerge.pair.source.shortcode)}>
+        { enrolleeMerge.pair.source.shortcode }
+      </Link> }
+
+    {mergeArrow}
+    { enrolleeMerge.pair.target &&
+      <Link to={studyEnvParticipantPath(studyEnvParams, enrolleeMerge.pair.target.shortcode)}>
+        { enrolleeMerge.pair.target.shortcode }
+      </Link> }
+    <table>
+      <thead>
+        <tr>
+          <th>task</th>
+          <th className="px-2">duplicate</th>
+          <th className="px-2">original</th>
+          <th className="px-2">Merge result</th>
+        </tr>
+      </thead>
+      <tbody>
+        { sortedTasks.map((taskMerge, index) =>
+          <MergeTaskView taskMerge={taskMerge} key={index}/>) }
+      </tbody>
+    </table>
+  </div>
+}
+
+function MergeTaskView({ taskMerge }: {taskMerge: MergeAction<ParticipantTask, object>}) {
+  const source = taskMerge.pair.source
+  const target = taskMerge.pair.target
+  return <tr>
+    <td>{renderTaskTitle(source, target)}</td>
+    <td className="px-2">{renderTaskSummary(source)}</td>
+    <td className="px-2">{renderTaskSummary(target)}</td>
+    <td>{renderResultSummary(taskMerge)}</td>
+  </tr>
+}
+
+function renderTaskSummary(task?: ParticipantTask) {
+  if (!task) {
+    return <span className="text-muted fst-italic">n/a</span>
+  }
+  return <Button variant="secondary" tooltip={`assigned ${instantToDateString(task.createdAt)}`}>
+    {TASK_STATUS_ICONS[task.status]}
+  </Button>
+}
+
+function renderResultSummary(taskMerge: MergeAction<ParticipantTask, object>) {
+  if (taskMerge.pair.source && taskMerge.pair.target) {
+    if (taskMerge.action === 'MOVE_SOURCE') {
+      return <span>keep both</span>
+    } else if (taskMerge.action === 'MERGE') {
+      return <span>merge</span>
+    } else if (taskMerge.action === 'MOVE_SOURCE_DELETE_TARGET') {
+      return <span>keep duplicate, delete original</span>
+    } else if (taskMerge.action === 'DELETE_SOURCE') {
+      return <span>keep original, delete duplicate</span>
```

Comment:
```suggestion
      return <span>keep duplicate</span>
    } else if (taskMerge.action === 'DELETE_SOURCE') {
      return <span>keep original</span>
```
nit: I think i prefer just using `keep` language, frees up a lot of whitespace and I find it easier to scan

Reply:
agree it's wordy, but I like emphasizing that something is getting removed.  Changed to "keep duplicate only"